spec/core/data_structures.md

```diff
@@ -289,8 +289,8 @@ and uses a different ordering of the fields.
 | Name      | Type                            | Description                             | Validation                               |
 |-----------|---------------------------------|-----------------------------------------|------------------------------------------|
 | Type      | [SignedMsgType](#signedmsgtype) | The type of message the vote refers to  | Must be `PrevoteType` or `PrecommitType` |
-| Height    | int64                           | Height in which the vote was provided.  | Must be > 0                              |
-| Round     | int64                           | Round in which the vote was provided.   | Must be > 0                              |
+| Height    | int64                           | Height in which the vote was provided.  | Must be >= 0                             |
```

Comment:
```suggestion
| Height    | int64                           | Height in which the vote was provided.  | Must be > 0                             |
```

Reply:
There is not validation for this type. But it can only be produced from votes, for which `Height > 0` is required. Fixing.